### DISCUSSION TITLE
googlechrome using the early stable release

### DISCUSSION THREAD

**Discussion by julian-jordan** [chocolatey-community/chocolatey-packages#2470] [General]
Seems like the package has switched to the early stable release. Will this package use that release going forward? Reason I asked is because there are security fixes added to the stable release on [May 13](https://chromereleases.googleblog.com/2024/05/stable-channel-update-for-desktop_13.html) that likely did not make it into this early stable release on [May 8th](https://chromereleases.googleblog.com/2024/05/early-stable-update-for-desktop.html)

**Comment by AdmiringWorm** [comment]
We are using their version API to detect new versions of Google Chrome.

Specifically, we are using the feed `https://versionhistory.googleapis.com/v1/chrome/platforms/win/channels/stable/versions` right now, so it most likely would be the early stable release.

We need to use it from that version, as there is only one URL to the binary files, meaning that as soon as Google releases a new version (as an early stable release), then the package starts failing immediately.

~If the security fix is not in the version released or not, I honestly have no idea. What I did see, is that there is a new version not yet picked up on the version API feed, this should be picked up later today automatically.~

EDIT: I had mixed up the version numbers, so there is no new version pending.

**Comment by julian-jordan** [comment]
Looks like Google have published 125.0.6422.60 to address these vulnerabilities, so all good now. Thanks for the quick response
